compiler/test/runner.re

```diff
@@ -29,6 +29,7 @@ let read_stream = cstream => {
 let compile = (~hook=?, name, prog) => {
   Config.preserve_config(() => {
     Config.include_dirs := [test_libs_dir, ...Config.include_dirs^];
+    Config.no_bulk_memory := true;
```

Comment:
Just for showing this is working (or not, in the case of the 2 failing tests). Will remove before merging.